examples/simple_timer.rs

```diff
@@ -43,6 +38,19 @@ impl Actor for TimerExampleActor {
         println!("Got a message: {:?} at {:?}", message, self.started_at.elapsed());
         Ok(())
     }
+
+    fn deadline_passed(&mut self, context: &mut Context<Self>, _deadline: Instant) {
+        context.myself.send(TimerMessage::Periodic).unwrap();
+
+        // A: Schedule one second from now (even if delayed); drifting tick.
```

Comment:
Could you please move this documentation and examples to `lib.rs`? I think it's crucial to understand how to use the `deadline` effectively.